Yoast/Tests/Commenting/CodeCoverageIgnoreDeprecatedUnitTest.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class SomeThing {
+
+	/**
+	 * Active (non-deprecated) function.
+	 */
+	function activeFunction() {}
+
+	/**
+	 * @deprecated x.x
+	 * @codeCoverageIgnore
+	 */
+	public /* don't get confused by this */ function correctDocblock() {
+		// Code
+    } // end function
+
+    public function functionWithoutDocblock {}
```

Comment:
please notice this line uses spaces for indentation, it should use a tab character